content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -7,10 +7,13 @@ title: Schedule GPUs
 
 {{% capture overview %}}
 
-Kubernetes includes **experimental** support for managing NVIDIA GPUs spread
+Kubernetes includes **experimental** support for managing AMD and NVIDIA GPUs spread
```

Comment:
k8s support for Nvidia GPUs isn't experimental. It is used in production today by many users.